Automattic/jetpack
# Admin Page: Update upgrade buttons Tracks events

#### Changes proposed in this Pull Request:

* This PR adds a Tracks event when an upgrade button is displayed on the At a Glance page on the WP Admin:
The event is named `jetpack_wpa_aag_upgrade_button_view`, and an additional property `feature` (e.g: `scan`) is passed along.
* It also updates the `JetpackBanner`(`components/banner`) in order to register the event only when the upgrade banner is mounted (not the connection banner).

#### Jetpack product discussion


Context: pbNhbs-18H-p2#comment-3911

#### Does this pull request change what data or activity we track or use?


Yes, it adds a Tracks event when viewing the upgrade buttons on the At a Glance page.

#### Testing instructions:






* Go to `/wp-admin/admin.php?page=jetpack#/dashboard`
* Make sure your current plan is one that allows upgrades (I'd recommend using Free to test this)
* Scroll down and make sure that you got an event fired for every feature-related Upgrade button you see
   * Scan, Backup, and Anti-spam under "Security"
   * Search and VideoPress under "Performance" (You have to click to activate VideoPress before seeing the Upgrade button)
* You can check if the event got fired by going to the Network tab in your developer console, and filtering by `t.gif`.
